src/components/QuizResult.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function QuizResult() {
```

Comment:
Why do we have a component just to render one line of text?

Reply:
Removed this component as It was not used anywhere.